.changeset/wild-gorillas-battle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/platform-core': patch
```

Comment:
need to include cli package as well

Reply:
cli package is in the other changeset file